examples/example_patchextraction.ipynb

```diff
@@ -0,0 +1,383 @@
+{
```

Comment:
<p>Can we add a message that the download is complete?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TIA-Lab/tiatoolbox/pull/100/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
Resolved in this commit https://github.com/TIA-Lab/tiatoolbox/pull/100/commits/bf66f2bedb92ada16fa7e5f51a39b420a32c8298